cmd/bootstrap/cmd/constraints.go

```diff
@@ -9,8 +9,7 @@ import (
 // Checks constraints about the number of partner and internal nodes.
 // Internal nodes must comprise >2/3 of consensus committee.
 // Internal nodes must comprise >2/3 of each collector cluster.
-func checkConstraints(partnerNodes, internalNodes []model.NodeInfo) {
-
+func checkConsensusConstraints(partnerNodes, internalNodes []model.NodeInfo) {
```

Comment:
agree about removing https://github.com/onflow/flow-go/blob/fdbd24ca32980e04c5acb1d61aca48f4ca5cdc23/cmd/bootstrap/cmd/constraints.go#L31-L39

But regarding the check https://github.com/onflow/flow-go/blob/fdbd24ca32980e04c5acb1d61aca48f4ca5cdc23/cmd/bootstrap/cmd/constraints.go#L17-L29 I think we still need this for method [`checkCollectionConstraints`](https://github.com/onflow/flow-go/blob/fdbd24ca32980e04c5acb1d61aca48f4ca5cdc23/cmd/bootstrap/cmd/constraints.go#L46). Otherwise, `checkCollectionConstraints` might conclude that we have enough collectors in each cluster, yet, they don't represent a supermajority, because we are ignoring the weight in our check.

Reply:
fixed in https://github.com/onflow/flow-go/pull/1371/commits/503e8eae901a323858140c152d7e2b3694e1307c and https://github.com/onflow/flow-go/pull/1371/commits/e011135fd2721458d04e7b84b928f296c2a9bb02